src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -111,7 +111,17 @@ def run(self, revision):
             logger.error('Mach static analysis failed: {}'.format(e.output))
             raise
 
-        issues = self.parse_issues(clang_output.decode('utf-8'), revision)
+        clang_output = clang_output.decode('utf-8')
+
+        # Dump raw clang-tidy output as a Taskcluster artifact (for debugging)
+        clang_output_path = os.path.join(
+            settings.taskcluster_results_dir,
+            '{}-clang-tidy.txt'.format(revision),
```

Comment:
You should use something else than `Revision.__str__` here, as this methods outputs a display name with spaces.
Maybe implement [__repr__](https://docs.python.org/3/reference/datamodel.html#object.__repr__)

Reply:
Done by refactoring `revision.build_diff_name` into a `revision.__repr__`.